### DISCUSSION TITLE
Display error messages from new_project.kts in red text instead of the default color

### DISCUSSION THREAD

**Discussion by ryan-conway** [nimblehq/android-templates#271] [RFCs]
When running `new_project.kts`, it runs multiple steps, including but not limited to:

- Verify arguments
- Initialize new project folder
- Build and run tests

Suppose one of these steps fails, it will print a message saying what went wrong:
<img width="827" alt="Screenshot 2565-08-05 at 15 59 12" src="https://user-images.githubusercontent.com/53168251/183042280-beeba766-19d8-4dd9-bc75-65a79d9e91db.png">

We could update the message to appear in red text so that the error will be more apparent if one occurs:
<img width="838" alt="Screenshot 2565-08-05 at 15 58 53" src="https://user-images.githubusercontent.com/53168251/183042782-33a7f29f-216d-4339-a4fe-7fe461356d62.png">

**Advantage?**
When an error occurs, it will be much more obvious to the end user that something went wrong.
Would also help mitigate the disadvantage on #268.

**Disadvantage?**
May not display correctly with the default Windows terminal without using external libraries. ([source](https://stackoverflow.com/a/5762502/7561225))

**Comment by toby-thanathip** [comment]
@ryan-conway Cool! 🤩 

I like the idea, but I prefer keeping the script simple by not requiring an external library.

So my vote is "No", and prefer going with emojis (ex: ❌ or 🔴)

**Reply by manh-t** [reply]
To make it simple, I agree to go with emojis 👍

**Reply by chornerman** [reply]
Agree 🚀

**Comment by toby-thanathip** [comment]
@nimblehq/android-chapter For those who voted "Yes", which approach do you guys prefer?
- ASCII
- External library

**Comment by luongvo** [comment]
I voted No. I don't see many CLIs showing error messages in red. We might spend unnecessary effort on this 🙏

**Comment by thiennguyen0196** [comment]
Displaying error text with a noticeable sign is a good approach, but using another library to achieve it might be unnecessary. So my vote for this RFC is `No` ⛔  but using emoji in text like @Tuubz 's suggestion looks good to me 👍

**Comment by toby-thanathip** [comment]
@ryan-conway Voting completed, I'll leave it up to you to resolve this discussion 🚀

**Reply by toby-thanathip** [reply]
@ryan-conway Just to be sure we're aligned, even though the voting says we won't change the text color, we will still create an issue to add an ❌ emoji to the error message, right? 🙇

**Reply by ryan-conway** [reply]
Yes, will create it today 👍🏻

**Comment by ryan-conway** [comment]
Issue created: #437
